websocket/frontend/src/todolist/TodoList.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react';
+import TodoForm from "./Todoform";
+import TodoItem from "./TodoItem";
+
+const apiURL = 'http://localhost:3001/api/v1/todos/';
+
+class TodoList extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      items: []
+    };
+
+    this.updateTodoList = this.updateTodoList.bind(this);
+  };
+
+  componentDidMount() {
+    this.getTasks();
+  };
+
+  getTasks() {
+    fetch(apiURL)
```

Comment:
Why this uses normal promise while the other file uses await async? 
Please keep it consistent. Choose one or the other but not both